bucket/microsoft-lts-jdk.json

```diff
@@ -42,4 +42,4 @@
         },
         "extract_dir": "jdk-$matchVer+$matchBuild"
     }
-}
+}
```

Comment:
Done

Reply:
> You should add New Line here.

I thought that was detected in our GitHub actions for PR's?

Reply:
Maybe, we can add a config with `pre-commit` that will format the files before committing. Because I didn't know that due to CRLF and no new line, my PR would fail. We can eliminate this scenario by using it.

Reply:
Yes, this was detected by tests, see report https://github.com/ScoopInstaller/Java/actions/runs/8611549974?pr=512

![image](https://github.com/ScoopInstaller/Java/assets/5832170/eb85737f-3535-466c-b4fd-c0cee846720a)

Adding a `pre-commit` hook may be helpful here.